manim/mobject/svg/svg_mobject.py

```diff
@@ -287,7 +287,7 @@ def get_all_childNodes_have_id(self, element):
         all_childNodes_have_id = []
         if not isinstance(element, minidom.Element):
             return
-        if element.hasAttribute("id"):
+        if element.hasAttribute("id") and element.tagName not in ("g", "defs"):
```

Comment:
@cobordism Could you tell me why this should be done?

Reply:
If I take a maths pdf/dvi and use dvisvgm to get an svg, then I can import it to manim. (This is the normal flow)
If I open the svg in inkscape and make a few tweaks and save, I can no longer import it to manim.
This line changes that.

In general the svg handling is supposed to be rewritten, but since this change is so small and doesn't hurt anyone... we figured I could sneak it in here.

Reply:
Can I confirm that this doesn't affect other classes which depend on this? Like `Text`?